tests/test_package.py

```diff
@@ -1,20 +1,33 @@
 import unittest
 from pathlib import Path
 
+import pytest
+
 import runhouse as rh
 
+extra_index_url = "--extra-index-url https://pypi.python.org/simple/"
+cuda_116_url = "--index-url https://download.pytorch.org/whl/cu116"
+
 
 def setup():
     pass
 
 
+@pytest.fixture
+def cluster(request):
+    return request.getfixturevalue(request.param)
```

Comment:
What does this do?

Reply:
this lets us parametrize over multiple fixtures - useful for running the same test to multiple hardware types. for example: `@pytest.mark.parametrize("cluster", ["v100", "k80", "a10g"], indirect=True)`